src/targeting/Find_targets.ml

```diff
@@ -97,7 +97,7 @@ type conf = {
   always_select_explicit_targets : bool;
   explicit_targets : Explicit_targets.t;
   (* osemgrep-only: option (see Git_project.ml and the force_root parameter) *)
```

Comment:
This comment may be old. I don't see a "force_root" parameter in Git_project.ml (or let me know if I missed it).

Reply:
Yes, you missed. I made the comment more precise.